mapeditor/CMakeLists.txt

```diff
@@ -132,5 +132,5 @@ install(TARGETS vcmieditor DESTINATION ${BIN_DIR})
 install(DIRECTORY icons DESTINATION ${DATA_DIR}/mapeditor)
 # Install icons and desktop file on Linux
 if(NOT WIN32 AND NOT APPLE)
-	install(FILES "vcmilauncher.desktop" DESTINATION share/applications)
+	install(FILES "vcmieditor.desktop" DESTINATION share/applications)
```

Comment:
```suggestion
	install(FILES "${CMAKE_SOURCE_DIR}/mapeditor/icons/mapeditor.64x64.png"   DESTINATION share/icons/hicolor/64x64/apps RENAME mapeditor.png)
	install(FILES "${CMAKE_SOURCE_DIR}/mapeditor/icons/mapeditor.48x48.png"   DESTINATION share/icons/hicolor/48x48/apps RENAME mapeditor.png)
	install(FILES "${CMAKE_SOURCE_DIR}/mapeditor/icons/mapeditor.32x32.png"   DESTINATION share/icons/hicolor/32x32/apps RENAME mapeditor.png)
	install(FILES "${CMAKE_SOURCE_DIR}/mapeditor/icons/mapeditor.256x256.png" DESTINATION share/icons/hicolor/256x256/apps RENAME mapeditor.png)
	install(FILES "vcmieditor.desktop" DESTINATION share/applications)
```

Reply:
And the last one missed file.
mapeditor.64x64.png

![mapeditor 64x64](https://user-images.githubusercontent.com/87084363/197348065-318541f0-8753-437a-abab-6e83c528d897.png)